src-built-in/components/myApps/src/components/AppActionsMenu.jsx

```diff
@@ -31,17 +31,16 @@ export default class AppActionsMenu extends React.Component {
 		this.onRemove = this.onRemove.bind(this);
 		this.setMenuRef = this.setMenuRef.bind(this);
 		this.deleteApp = this.deleteApp.bind(this);
-		this.handleClickOutside = this.handleClickOutside.bind(this);
 	}
 
 	componentDidMount() {
-		document.addEventListener("mousedown", this.handleClickOutside);
-		finsembleWindow.addEventListener("blurred", this.handleClickOutside);
+		document.addEventListener("mousedown", this.handleClickOutside.bind(this));
+		finsembleWindow.addEventListener("blurred", this.handleClickOutside.bind(this));
 	}
 
 	componentWillUnmount() {
-		document.removeEventListener("mousedown", this.handleClickOutside);
-		finsembleWindow.removeEventListener("blurred", this.handleClickOutside);
+		document.removeEventListener("mousedown", this.handleClickOutside.bind(this));
+		finsembleWindow.removeEventListener("blurred", this.handleClickOutside.bind(this));
```

Comment:
This won't remove the bound listener, but instead try to unbind a new function. The binding has to be cached outside of the event listener calls as it was before.

It seems like if this is the solution then we have an architectural smell that needs to be handled.

Reply:
You're absolutely right, I don't know what I was thinking.

This did solve the error though, which as you said is probably indicative of a larger problem.

I'll circle back on this and see if I can find a better solution that also solves the error message